recipes/pathogist/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.2.2" %}
+package:
+  name: pathogist
+  version: {{ version }}
+
+source:
+  url: https://github.com/WGS-TB/PathOGiST/archive/v{{ version }}.tar.gz
+  sha256: 6691d1e08648ebe7aa743a42df51d64180824dbbd2959e449f3b11ebc39951d4
+
+build:
+  number: 1
+  skip: True # [osx]
+
+requirements:
+  host:
+    - python ==3.5
```

Comment:
Does this really only work with python 3.5? We're going to stop building for that rather soon I expect.

Reply:
It has only been tested on Python 3.5. The next version will be tested for newer versions of Python.